c2rust-analyze/src/borrowck/type_check.rs

```diff
@@ -369,7 +396,7 @@ impl<'tcx> TypeChecker<'tcx, '_> {
                     */
                     assert_eq!(expect_ty.args.len(), 0, "Generic types not yet supported.");
 
-                    let adt_def = self.tcx.adt_def(adt_did);
+                    let adt_def = self.acx.tcx().adt_def(adt_did);
```

Comment:
In places where the `TyCtxt` is used multiple times, I'll usually put a `let tcx = self.acx.tcx();` or similar at the top of the function, so the use sites can avoid a bit of boilerplate (there is only one `tcx` - it doesn't really matter exactly how you got hold of it)

Reply:
60be3579